src/dirs.rs

```diff
@@ -42,6 +31,27 @@ impl BatProjectDirs {
         })
     }
 
+    pub fn get_cache_dir() -> Option<PathBuf> {
+        // on all OS prefer BAT_CACHE_PATH if set
+        let cache_dir_op = env::var_os("BAT_CACHE_PATH")
+            .map(PathBuf::from)
+            .filter(|p| p.is_absolute());
```

Comment:
Why this constraint? To help users shooting themselves in the foot? If so, we shouldn't silently fall back to the standard cache dir in this case, but rather print a warning.

If there is no strong reason for doing so, I would just remove this `.filter(…)`. If a user wants to quickly test something, using a relative cache path might even be useful.

Reply:
There wasn't a specific reason for adding that constraint. It was a lazy logic copy from other place in this file doing that. Thinking about it now, it makes much more sense to not require absolute paths here. I'll remove it.